mm2src/kdf_walletconnect/src/lib.rs

```diff
@@ -358,10 +359,11 @@ impl WalletConnectCtxImpl {
             .await?;
 
         let (tx, rx) = oneshot::channel();
+        // insert request to map with a reasonable expiration time of 30 minutes
```

Comment:
ummm... isn't 30 mins veeery long?

Reply:
WalletConnect doesn't limit timeout it all depends on the timeout we want to set for each request.
I agree 30minsis a very long expiration time.

reduced to 5 mins